packages/aws-rfdk/lib/deadline/README.md

```diff
@@ -358,7 +358,7 @@ repository.configureClientInstance({
 ### Configuring Deadline Secrets Management on the Repository
 
 By default, [Deadline Secrets Management](https://docs.thinkboxsoftware.com/products/deadline/10.1/1_User%20Manual/manual/secrets-management/deadline-secrets-management.html) is enabled on the `Repository`.
-when using Deadline 10.1.19 or higher. RFDK will create administator credentials for Deadline Secrets Management, store them in AWS Secrets Manager, and configure the Deadline Repository with those credentials.
+when using Deadline 10.1.19 or higher. RFDK is able to create administator credentials for Deadline Secrets Management, store them in AWS Secrets Manager, and configure the Deadline Repository with those credentials. When RFDK generates the admin credentials for you, it puts them in the same Stack as your database so that their lifecycles are in sync and you cannot accidentally delete the credentials that the database is configured to use. If your database is imported into your RFDK app rather than being generated by it, RFDK will not allow your app to have credentials generated automatically and you'll need to create your own AWS Secret with the credentials you'd like to use and import it.
```

Comment:
```suggestion
when using Deadline 10.1.19 or higher. RFDK is able to create administrator credentials for Deadline Secrets Management, store them in AWS Secrets Manager, and configure the Deadline Repository with those credentials. When RFDK generates the admin credentials for you, it puts the `Secret` in the same `Stack` as your database so that their lifecycles are in sync, preventing the admin credentials from being destroyed by RFDK while the database is still in use. If your database is imported into your RFDK app rather than being generated by it, RFDK will not allow your app to have credentials generated automatically and you'll need to import your own AWS Secret with the credentials you'd like to use.
```

Couple things:
1. Looks like there's a typo in "administator" that I missed from the previous PR 😞 could you fix it?
2. I think we can be more specific about the reason we tie the lifecycle of the Secret to the database.